source/WB-CAN-Firmware.c

```diff
@@ -157,7 +165,7 @@ int main(void) {
 		512,
 		NULL,
 		0,
-		NULL)) != pdPASS) {
+		&actuator_task)) != pdPASS) {
```

Comment:
This is great. The reason we did this was to allow our main control task to reference this task using this handle, specifically for Suspending and Resuming Tasks.

Line 241:
`vTaskSuspend(actuator_task);`